java/pom.xml

```diff
@@ -52,14 +52,17 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-release-plugin</artifactId>
-                <version>2.2.1</version>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Release plugin wasn't really used for anything. Jar plugin allows us to generate an additional file named hebmorph-lucene.jar automatically.